VitruvioHost/Plugins/Vitruvio/Source/VitruvioEditor/Private/VitruvioEditorModule.cpp

```diff
@@ -15,28 +15,40 @@
 
 #include "VitruvioEditorModule.h"
 
+#include "ChooseRulePackageDialog.h"
 #include "RulePackageAssetTypeActions.h"
+#include "VitruvioActor.h"
 #include "VitruvioComponentDetails.h"
+#include "VitruvioCooker.h"
 
+#include "Algo/AllOf.h"
+#include "Algo/AnyOf.h"
 #include "AssetToolsModule.h"
 #include "Core.h"
+#include "Editor/LevelEditor/Public/LevelEditor.h"
+#include "Framework/Notifications/NotificationManager.h"
 #include "IAssetTools.h"
 #include "Modules/ModuleManager.h"
+#include "Widgets/Notifications/SNotificationList.h"
 
 #define LOCTEXT_NAMESPACE "VitruvioEditorModule"
 
-#include "Algo/AnyOf.h"
-#include "ChooseRulePackageDialog.h"
-#include "Editor/LevelEditor/Public/LevelEditor.h"
-#include "Framework/Notifications/NotificationManager.h"
-#include "VitruvioActor.h"
-#include "Widgets/Notifications/SNotificationList.h"
-
 namespace
 {
 
 bool IsViableVitruvioActor(AActor* Actor)
 {
+
+	if (Cast<AVitruvioActor>(Actor))
+	{
+		return false;
+	}
```

Comment:
The naming of this function is a bit confusing. Should either be commented or renamed.